cibyl/orchestrator.py

```diff
@@ -327,4 +327,5 @@ def query_and_publish(self, output_style: str = "colorized",
                 environment=env,
                 style=output_style,
                 query=get_query_type(**self.parser.ci_args),
-                verbosity=self.parser.app_args.get('verbosity'))
+                verbosity=self.parser.app_args.get('verbosity'),
+                complete=self.parser.app_args.get('complete', None))
```

Comment:
If 'complete' is not in the arguments, what you would like to do is pass 'False', as that is the argument's default value. Passing 'None' will most likely end in an error somewhere.

Also, verbosity should default to '0'. That is a mistake we have there too.